static/scenes/railroad/js/scene.js

```diff
@@ -44,6 +44,10 @@ class PlaceholderScene {
       radius * Math.sin(angle),
     );
   }  
+
+  getScene() {
```

Comment:
At the moment I don't think there's a strong reason to add a getter vs just treating the property as public. Let's just use the scene variable here.